docs/getting-started/quickstart.md

```diff
@@ -192,13 +187,11 @@ if __name__ == "__main__":
         entrypoint="my_gh_workflow.py:repo_info",
     ).deploy(
         name="my-first-deployment",
-        work_pool_name="my-managed-pool",
```

Comment:
Actually a content from line 186 - I suggest we point out more clearly that here they're using code we've stored in GitHub.